torch/jit/_script.py

```diff
@@ -879,8 +877,18 @@ def _script_pdt(obj, optimize=None, _frames_up=0, _rcb=None, example_inputs: Opt
     if monkeytype_trace:
         monkeytype_config = JitTypeTraceConfig(type_trace_db)
         with monkeytype_trace(monkeytype_config):
-            for example_input in example_inputs:  # type: ignore[union-attr]
-                obj(*example_input)
+            for example_input in example_inputs:    # type: ignore[union-attr]
```

Comment:
I think this `union-attr` lint error will go away if you check `example_inputs` is not `None` before iterating through it. This is an actual bug, please don't suppress lint warnings unless you have a good reason to do so. Same for line 854